apps/tup-ui/index.html

```diff
@@ -10,24 +10,9 @@
         baseUrl: 'https://dev.tup-services.tacc.utexas.edu',
       };
     </script>
-    <style>
-      body {
-        margin: 0px 0px 0px 0px;
-        left: 0px;
-        right: 0px;
-        top: 0px;
-        bottom: 0px;
-      }
-      html,
-      body,
-      #root,
-      #root > div {
-        height: 100%;
-      }
-    </style>
```

Comment:
ℹ️ Moved to [apps/tup-ui/src/main.css](https://github.com/TACC/tup-ui/pull/47/files#diff-d3a034d2bd8e0eb8ffb07a28f730fce5185cfe7407059ffe2621fecd02172eb9) and [apps/tup-ui/src/pages/Login/Layout.global.css](https://github.com/TACC/tup-ui/pull/47/files#diff-3797b2daa362806d00e00c64497a183229c685987c833099a614bc9b04b044a9).